### PR TITLE
Changelog v3

### PR DESCRIPTION
**Pull request by varas**
Changelog for version 3.0.0-beta

### Author

- [x] add a risk label after carefully considering the "blast radius" of your changes
- [x] describe the _intent_ of your changes in the description. don't just rewrite your code in prose
- [x] assign at least one reviewer

### Reviewer

- [ ] review code for readability
- [ ] verify that high risk behavior changes are well tested
- [ ] check license for any new external dependency
- [ ] ask questions about anything that isn't clear and obvious
- [ ] approve the PR when you consider it's good to merge

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,20 +4,24 @@ All notable changes to this project will be documented in this file.
...
-### Fixed
-- github.com/Sirupsen/logrus is now github.com/sirupsen/logrus since v1.0.0.
+- Protocol v2 (remote entities) support
+- Concurrency support (via `Synchronized` *option*)
+- Configurable `Logger`
+- Configurable output `Writer`
+- Configurable `Storer`
+- New packages `inventory`, `event` with proper constructors 
+- Integration parametrized creation via optional `Option`s
+### Changes *(breaking changes)*
```
Comment (smoya) [inline CHANGELOG.md]: Missing the `### Removed` section. We removed support for `v1` right?

File: CHANGELOG.md [hunk 2]
```diff
@@ -34,11 +38,13 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 - **Breaking change**: Update `jmx.Query` function adding timeout argument
 - Improve documentation
 
+
```
Comment (smoya) [inline CHANGELOG.md]: Why those extra lines?

Comment (varas) [reply CHANGELOG.md]: Just readability between major versions

File: CHANGELOG.md (hunk 1 above)
Comment (varas) [reply CHANGELOG.md]: Indeed, added

Comment (smoya) [reply CHANGELOG.md]: Sorry But I forgot to mention that you must add a blank line between the previous list and this title. Some Markdown editors does not render this properly and mix them. You know, markdown stuff...

screen shot 2018-04-30 at 12 44 17 pm

**Pull request merged**